File: Deliverables/BizBong/app/src/main/java/gamesoftitalia/bizbong/entity/Classificato.java
package gamesoftitalia.bizbong.entity;

/**
 * Created by GameSoftItalia on 09/01/2017.
 */

public class Classificato extends Profilo {

    private int punteggio;

    public Classificato(String nickname, String img_profilo, int punteggio) {
        super(nickname, img_profilo);
        this.punteggio = punteggio;
    }

    public void setPunteggio(int punteggio){
        this.punteggio = punteggio;
    }

    public int getPunteggio(){
        return punteggio;
    }
}

File: Deliverables/php/Entity/Classificato.php
<?php
	//**********************************************************************************************
    // Creazione dell' oggetto Classificato eredità Profilo che implementa JsonSerializable per essere inviato tramite json
    // Variabili: nickname, email, img_profilo, statistica, punteggio
    // jsonSerialize() --> utilizzato per la conversione a json
    //**********************************************************************************************
	
    include ".../Entity/Profilo.php";
    
    class Classificato extends Profilo implements JsonSerializable {
    	private $punteggio;
        
        public function Classificato($nickname, $img_profilo, $punteggio) {
        	parent::setNickname($nickname);
            parent::setImgProfilo($img_profilo);
            $this->punteggio = $punteggio;
        }
        
        public function getPunteggio(){
        	return $this->punteggio;
        }
        
        public function setPunteggio($punteggio){
        	$this->punteggio = $punteggio;
        }
        
        public function jsonSerialize() {
        	$obj = parent::jsonSerialize();
        	$obj["punteggio"] = $this->punteggio;
        	return $obj;
    	}
    }
?>